commons/passive/carl/FibaroRoom.avsc

```diff
@@ -0,0 +1,11 @@
+{
+ "namespace": "org.radarcns.active.carl",
+ "type": "record",
+ "name": "Room",
+ "doc": "A building Room that has Fibaro Devices assigned to it",
+ "fields": [
+     {"name": "id", "type": "double", "doc": "Unique ID for a Room"},
+     {"name": "name", "type": "string", "doc": "Room's Name"},
+     {"name": "sectionId", "type": ["null","double"], "doc": "Section's unique ID where the Room belongs to"}]
```

Comment:
please add a `time` field

Reply:
please add a `timeReceived` field to represent when this record was received.

Reply:
use `int` for sectionId?

Reply:
@yatharthranjan fibaro_room and fibaro_section do not have any time related data. Are these fields a must-have ?

@blootsvoets we used doubles, since we used these for similar fields on BankingApp. But int seems more suitable.

Reply:
These fields are a must-have. If this topic only contains device metadata, you could add this particular schema to the `commons/monitor/fibaro` instead. Please consider https://github.com/RADAR-base/RADAR-Schemas/blob/master/commons/monitor/application/application_device_info.avsc as an example. In that case only a `time` field is needed, not `timeReceived`. And `time` in that case is the time that you added this device (sent the record). If you choose this route, you'll eventually have to add two specifications: 1 for `monitor` metadata like this, and one for `passive` actual data.

Reply:
Since the the names can change I'm going to add the time and timeReceived fields.

So time field is used to represent when our platformt got the data and timeReceived is when we sent this data to RADAR, correct ?

Reply:
That should be fine. The `time` field should encode when the name change was made, then `timeReceived` should be when you created this record to send it to radar-base.